lte/gateway/python/integ_tests/s1aptests/BUILD.bazel

```diff
@@ -2213,11 +2222,20 @@ pytest_test(
 )
 
 pytest_test(
-    name = "test_attach_detach_with_non_nat_dhcp",
+    name = "test_attach_detach_with_non_nat_dhcp_multi_ue",
     size = "large",
-    srcs = ["test_attach_detach_with_non_nat_dhcp.py"],
+    srcs = ["test_attach_detach_with_non_nat_dhcp_multi_ue.py"],
     imports = [LTE_ROOT],
-    tags = TAG_NON_SANITY_TEST,
+    tags = TAG_NON_SANITY_TEST + TAG_TRAFFIC_SERVER_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_attach_detach_with_non_nat_dhcp_multi_ue_looped",
+    size = "large",
```

Comment:
Do the selected test sizes make sense for these tests?
![Screenshot from 2022-12-20 13-25-07](https://user-images.githubusercontent.com/34488763/208666856-70634fb6-35d7-47e5-a74d-be9ce69424fb.png)

Reply:
Yes, this test can take over five minutes to run and the other one is also of similar duration.